src/WWTMVC5/Controllers/DefaultController.cs

```diff
@@ -20,11 +22,14 @@ public class DefaultController : ControllerBase
         /// Instance of Queue Service
         /// </summary>
         private INotificationService _notificationService;
+        private readonly ILogger<DefaultController> _logger;
+
         public DefaultController(IProfileService profileService, ICommunityService communityService, INotificationService queueService)
             : base(profileService)
         {
             _communityService = communityService;
             _notificationService = queueService;
+            _logger = UnityConfig.Container.Resolve<ILogger<DefaultController>>();
```

Comment:
You should be able to request ILogger<DefaultController> in the constructor

Reply:
<>

Reply:
Ah, gotcha (I think). I'll merge this now and see what I can figure out, and change this in the inevitable next round of work.